ipa-core/src/ff/boolean_array.rs

```diff
@@ -95,6 +101,8 @@ macro_rules! boolean_array_impl_small {
 
         // TODO(812): remove this impl; BAs are not field elements.
```

Comment:
It will be resolved by the two branches we looked at on Friday:

https://github.com/andyleiserson/ipa/compare/bit-vectorization...andyleiserson:ipa:u128?expand=1
https://github.com/andyleiserson/ipa/compare/u128...andyleiserson:ipa:field-cleanup?expand=1